CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
+3. Once you are in your `quire-docs` folder, run the following commands:
+```
+quire install
+quire preview
+```
+4. Go to [http://localhost:1313](http://localhost:1313) the project as you work on it.
 
-The Quire documentation is hosted in a separate repository along with the website, and can be found [here](https://github.com/thegetty/quire-docs). The Quire documentation is continuously being updated for clarity and completeness, but it’s not always easy to keep up with the pace of Quire’s development!
+**Open Your Project in GitHub Desktop**
+1. At the top right, click "Current Repository" and choose the `quire-docs` repo.
 
-If there’s a section you think it missing or could be improved, we’d love for your help. And the documentation is itself a Quire site, so it should be familiar to work in.
+**Create a New Branch in GitHub Desktop**
+1. Navigate to where it says “Current Branch” at the top of your GitHub Desktop window.
+2. Click the drop-down arrow and select “new branch” to the right of the Filter box.
+3. Name your branch.
+4. Select “main” where it says “Create branch based on.” This means you are creating a new branch that runs parallel to the main project.
 
-[Propose a new section or edit](https://github.com/thegetty/quire-docs/issues)<br />
-[Find an existing issue to work on](https://github.com/thegetty/quire-docs/issues)<br />
-[Read the Quire Documentation Style Guide](https://github.com/thegetty/quire-docs/wiki/Quire-Website-&-Documentation-Style-Guide)
+**Open Project File in Atom**
+1. Open Atom
+2. Click File > Open
+3. Navigate to your home directory and your select `quire-docs` folder.
+5. In Atom, expand the contents folder to see the editable markdown files and look for the file you'd like to make your changes in.
 
-### Translate the Documentation
+**Save Changes in Atom**
+1. To save changes, hit File > Save or Command-S (Mac) or Control-S (PC).
+2. If working on multiple markdown files, make sure to save each file individually. You will see a little blue dot at the top of the page if there are any unsaved changes.
+3. Double check that your changes look right by previewing at [http://localhost:1313](http://localhost:1313).
 
-The Quire community is global and we hope to be able to continue to expand access to Quire, including to non-English speakers. Though we haven’t yet done any translation work on any of the documentation or materials, if that’s something you’re interested in doing, we’d love to talk to you.
+**Make Commits in GitHub Desktop**
+1. At the bottom left, above where it says description, write a short name for your commit. You have the option to include a description as well.
+2. Press the blue button that says “Commit to [branch name].”
 
-[Volunteer to do some translation](https://github.com/thegetty/quire-docs/issues)
+**Push Changes in GitHub**
+1. After you have made all your commits, click on the "Publish Branch" and then “Push Origin” button at the top right.
```

Comment:
there might also be a "Publish Repository" step in here, or it might say "Publish Repository" instead of "Publish Branch" if they cloned the project locally but have not yet pushed it to their GitHub account. Not 100% sure of this though.

Reply:
Actually, come to think of it, instead of cloning, I think we want them to Fork the repository which would change some things here, and in the "Submit Your Contribution" section. Do you want to try this yourself with our @quire_user account?